lib/evmone/advanced_instructions.cpp

```diff
@@ -269,6 +269,7 @@ constexpr std::array<instruction_exec_fn, 256> instruction_implementations = [](
     table[OP_DATALOADN] = op_undefined;
     table[OP_DATASIZE] = op_undefined;
     table[OP_DATACOPY] = op_undefined;
+    table[OP_RETURNDATALOAD] = op_undefined;
```

Comment:
Also for should be fine to have it in Advanced.

Reply:
But only if it is supported in legacy, right? If it isn't then I suppose it would be on the same boat as rest of eof opcodes. Let's handle legacy-or-not first.